sys/auto_init/auto_init.c

```diff
@@ -20,6 +20,11 @@
 
 #include "auto_init.h"
 
+#ifdef MODULE_TLSF_MALLOC
+#include "tlsf-malloc.h"
+static uint8_t _tlsf_pool[TLSF_MALLOC_HEAPSIZE];
```

Comment:
On platforms other than malloc this can be derived from `_sheap` and `_eheap` linker symbols, which has the advantage of automatically using all available heap.